x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -119,8 +119,9 @@ export class ExecutionHandler<
       string,
       Alert<State, Context, ActionGroupIds> | Alert<State, Context, RecoveryActionGroupId>
     >
-  ): Promise<void> {
+  ): Promise<ThrottledActions> {
```

Comment:
**nit:** can we make this function return something more descriptive?

```suggestion
  ): Promise<{ throttledActions: ThrottledActions }> {
```

Reply:
Done in [#3d137a1](https://github.com/elastic/kibana/pull/147360/commits/3d137a13893d66c7b29ac93e20446f4fefbba12c)